depends/patches/native_biplist/sorted_list.patch

```diff
@@ -1,6 +1,6 @@
---- a/biplist/__init__.py	2014-10-26 19:03:11.000000000 +0000
-+++ b/biplist/__init__.py	2016-07-19 19:30:17.663521999 +0000
-@@ -541,7 +541,7 @@
+--- a/biplist/__init__.py
++++ b/biplist/__init__.py
```

Comment:
There is no problem even without this hunk 8) It is there to apply cleanly. No other purpose. And as such, it is useless IMO.